### PR TITLE
Update main README

### PR DESCRIPTION
**Pull request by AlmogCohen**
fix typo in main README and specify the arguments of the admin creation command

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,13 +64,13 @@ You'll probably need a default user to manage your application with, so you can
 
 Or you can pass some arguments:
    
-    $ zappa manage <stage> create_admin_user one two three
+    $ zappa manage <stage> create_admin_user <username> <email> <password>
```
Comment (FlipperPA) [inline README.md]: This isn't necessarily the case, if using a custom user model, is it?

File: README.md [hunk 2]
```diff
@@ -64,13 +64,13 @@ You'll probably need a default user to manage your application with, so you can
...
 
 This will internally make this call:
 
-```python
-User.objects.create_superuser('one', 'two', 'three')
-```
+    ```python
+    User.objects.create_superuser('username', 'email', 'password')
```
Comment (FlipperPA) [inline README.md]: Same comment as above; re: custom user models.

**Comment by AlmogCohen** [comment]
Make sense.

If I undo these two we'll get the first typo in....right?

Thanks for the awesome repo

**Comment by FlipperPA** [comment]
@AlmogCohen Absolutely, thank you for the good catch. :)

**Comment by AlmogCohen** [comment]
What do you say about that? I added the "default behavior" section since I had to look into the code to understand what "one two three" would actually mean. What do you think? can we keep that?

**Comment by AlmogCohen** [comment]
please merge, small typo

**Comment by AlmogCohen** [comment]
plz

**Comment by AlmogCohen** [comment]
No rush :) It is really trivial though

**Pull request merged**